### ISSUE TITLE
Test suite: ImportError (cannot import name 'PY3')

### ISSUE THREAD

**Issue by bowlofeggs** [Cornices/cornice.ext.sphinx#6]
```
======================================================================
ERROR: Failure: ImportError (cannot import name 'PY3')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/rbarlow/devel/cornice.ext.sphinx/.tox/py34/lib/python3.4/site-packages/nose/failure.py", line 39, in runTest
    raise self.exc_val.with_traceback(self.tb)
  File "/home/rbarlow/devel/cornice.ext.sphinx/.tox/py34/lib/python3.4/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/rbarlow/devel/cornice.ext.sphinx/.tox/py34/lib/python3.4/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/rbarlow/devel/cornice.ext.sphinx/.tox/py34/lib/python3.4/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/rbarlow/devel/cornice.ext.sphinx/.tox/py34/lib64/python3.4/imp.py", line 245, in load_module
    return load_package(name, filename)
  File "/home/rbarlow/devel/cornice.ext.sphinx/.tox/py34/lib64/python3.4/imp.py", line 217, in load_package
    return methods.load()
  File "<frozen importlib._bootstrap>", line 1220, in load
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1471, in exec_module
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  File "/home/rbarlow/devel/cornice.ext.sphinx/tests/__init__.py", line 7, in <module>
    from cornice_sphinx import rst2html, ServiceDirective
  File "/home/rbarlow/devel/cornice.ext.sphinx/cornice_sphinx/__init__.py", line 16, in <module>
    from cornice.util import to_list, is_string, PY3
ImportError: cannot import name 'PY3'
```

**Comment by bowlofeggs** [comment]
It appears that Travis is also experiencing this error, as I see that #5 also seems to have hit it.

**Comment by bowlofeggs** [comment]
I believe #4 was written to address this issue.

**Comment by naoko** [comment]
I'm experiencing the same issue and #4 does resolve the issue. Anyway the fix can be merged?

**Comment by Natim** [comment]
Fixed in 0.2
